haystack/core/pipeline/base.py

```diff
@@ -179,7 +180,17 @@ def from_dict(
 
                 # Create a new one
                 component_class = component.registry[component_data["type"]]
-                instance = component_from_dict(component_class, component_data, name, callbacks)
+
+                try:
+                    instance = component_from_dict(component_class, component_data, name, callbacks)
+                except Exception as e:
+                    msg = (
```

Comment:
how about adding a link to the recent changelogs?

Reply:
Added.